web/crux/prisma/migrations/20230612084035_add_default_value_to_user/migration.sql

```diff
@@ -0,0 +1,9 @@
+/*
+  Warnings:
+
+  - Made the column `user` on table `ContainerConfig` required. This step will fail if there are existing NULL values in that column.
+
+*/
+-- AlterTable
+ALTER TABLE "ContainerConfig" ALTER COLUMN "user" SET NOT NULL,
+ALTER COLUMN "user" SET DEFAULT -1;
```

Comment:
As per the warning above the migration will fail with null values in the db. You need to separate the two alter column commands, and update all the null values, before applying the second one.